headers.js

```diff
@@ -6,6 +6,8 @@ const cspMeta = Object.entries({
   'connect-src': [
     "'self'",
     'data:',
+    // Explicitly whitelist our KeepKey versions file
+    'https://bafybeied24gc2ipvlxdbs4v676dwho2l5aafmngrleic3do2czdvgb546u.ipfs.dweb.link/keepKey.json',
```

Comment:
where did this URL come from?

please document in a comment details about where to get more information about how to find/generate this URL.

Why are we not just pulling the firmware list directly from GitHub?